src/components/structures/MatrixChat.js

```diff
@@ -1561,7 +1561,16 @@ export default React.createClass({
         } else if (screen.indexOf('room/') == 0) {
             const segments = screen.substring(5).split('/');
             const roomString = segments[0];
-            const eventId = segments[1]; // undefined if no event id given
+            let eventId = segments.splice(1).join("/"); // empty string if no event id given
```

Comment:
would `screen.substring(5).split('/', 1)` be shorter / clearer / faster?

Reply:
This wouldn't fix the problem:
![image](https://user-images.githubusercontent.com/1190097/55102420-f7d33100-508b-11e9-95b9-2c12b2b72146.png)

(note: `part2` goes missing)

Reply:
oh yes, I forgot / failed to realise that js's string.split is different and less useful than string.split in any other language :/

Reply:
yup :|